assets/app/views/images.html

```diff
@@ -10,30 +10,55 @@
           <em>{{emptyMessage}}</em>
         </div>
       </div>
-      <div ng-repeat="imageStream in imageStreams">
-        <div class="tile" ng-repeat="tag in imageStream.status.tags">
-          <div>
-            <span>{{imageStream.metadata.name}}:{{tag.tag}}</span>
-            <span ng-if="!tag.items.length"> &mdash; <em>No images for this tag</em></span>
-            <span ng-if="tag.items.length && tag.items[0].image">
-              <span> &mdash; </span>
-              <short-id id="{{tag.items[0].image | imageName}}"></short-id>
-              <span> created </span>
-              <relative-timestamp timestamp="tag.items[0].created"></relative-timestamp>
+      <div ng-repeat="imageStream in imageStreams track by (imageStream | uid)" class="tile">
+        <h2>
+          {{imageStream.metadata.name}}
+        </h2>
+        <div ng-if="imageStream.spec.dockerImageRepository" class="small">Docker image repository: {{imageStream.spec.dockerImageRepository}}</div>
+        <!-- Print out missing status tags with a warning. If the dockerRepositoryCheck annotation
+             is present, but status tag is missing, tag could not be resolved. Otherwise, it's simply
+             not synced yet. -->
+        <div ng-repeat="specTag in missingStatusTagsByImageStream[imageStream.metadata.name] | orderBy : 'name' track by specTag.name"
+             style="margin-top: 10px;">
+          <div ng-if="imageStream | annotation : 'openshift.io/image.dockerRepositoryCheck'">
+            <span class="pficon-layered" style="margin-right: 5px;"
+                  ng-attr-title="{{imageStream | annotation : 'openshift.io/image.dockerRepositoryCheck'}}">
+              <span class="pficon pficon-warning-triangle"></span>
+              <span class="pficon pficon-warning-exclamation"></span>
             </span>
+            {{specTag.name}}
+            &mdash;
+            <span class="small">Tag could not be resolved.</span>
+          </div>
+          <div ng-if="!(imageStream | annotation : 'openshift.io/image.dockerRepositoryCheck')">
+              {{specTag.name}}
+              &mdash;
+              <span class="small text-muted">Not synced.</span>
```

Comment:
`Not yet synced`? No period (it's a fragment), and "yet" implies that it should happen without their intervention